core/templates/pages/admin-page/admin-page.module.ts

```diff
@@ -16,11 +16,12 @@
  * @fileoverview Module for the admin page.
  */
 
-import { APP_INITIALIZER, NgModule, StaticProvider } from '@angular/core';
+import { APP_INITIALIZER, ErrorHandler, NgModule, StaticProvider } from '@angular/core';
 import { FormsModule } from '@angular/forms';
 import { BrowserModule, HAMMER_GESTURE_CONFIG } from '@angular/platform-browser';
 import { downgradeComponent } from '@angular/upgrade/static';
-import { HttpClientModule } from '@angular/common/http';
+// eslint-disable-next-line oppia/disallow-httpclient
```

Comment:
Can you add an explanation here for why we need this disable in this case?

Reply:
This is a standard practice, we should disable this lint check for module files.

Reply:
The comment below also addresses this.